packages/backend-function/src/factory.ts

```diff
@@ -220,6 +248,33 @@ class FunctionFactory implements ConstructFactory<AmplifyFunction> {
 
     return this.props.runtime;
   };
+
+  private resolveSchedule = () => {
```

Comment:
We should pull all of this out into a separate `CronParser` that the function factory uses. All of this shouldn't be in the factory itself